python/paddle/fluid/tests/unittests/test_eager_deletion_padding_rnn.py

```diff
@@ -472,18 +434,32 @@ def encoder_static(input_embedding, len=3, init_hidden=None,
     layers.assign(input=last_hidden, output=init_hidden)
 
     feeding_list = ['x', 'y', 'init_hidden', 'init_cell']
-    if use_py_reader:
-        return loss, last_hidden, last_cell, feeding_list, py_reader
-    else:
-        return loss, last_hidden, last_cell, feeding_list
+    return loss, last_hidden, last_cell, feeding_list
 
 
 class PaddingRNNTestBase(unittest.TestCase):
     def setUp(self):
         self.reader = Reader()
         self.device_count = 1
 
-    def prepare_program(self, config, parallel=True):
+        # Default exec_strategy
```

Comment:
Can we delete the comment "Default exec_strategy"?

Because you initialize a default exec_strategy but set some values. Then it is not default, I think. Same as "Default build_strategy"

Reply:
I mean this is the default build_strategy and exec_strategy used for this program. I will refine the annotate to make it not ambiguous.